projects/plugins/jetpack/extensions/blocks/send-a-message/whatsapp-button/edit.js

```diff
@@ -200,57 +60,20 @@ export default function WhatsAppButtonEdit( { attributes, setAttributes, classNa
 		<div className={ getBlockClassNames() }>
 			{ ToolbarGroup && (
```

Comment:
Do we still need this check for `ToolbarGroup`? Was this to support an older version of WP?

It looks like most other blocks in Jetpack don't make this check. This would only tidy the code a tiny bit so it's fine, just more for my understanding.

Reply:
Yeh it seems like the only reason to have this would be for older versions of gutenberg -  @apeatling do you know if that is the case and if it is still needed?

Reply:
Just thought I'd look this one up. I think the check isn't needed any more, the `ToolbarGroup` component was added in https://github.com/wordpress/gutenberg/pull/18534 in GB 7.0, and from looking at the [Versions in WordPress doc](https://developer.wordpress.org/block-editor/handbook/versions-in-wordpress/), that was included in WP 5.4.0. According to the `readme.txt` the required WP version for Jetpack is currently [WP 5.5](https://github.com/Automattic/jetpack/blob/ece061248c19609e58fd67ba3ff69d850e6780e6/projects/plugins/jetpack/readme.txt#L5), so we should be good :)

Reply:
thanks for doing that research @andrewserong - have removed that check now